package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "analyze": "webpack --optimize-minimize; webpack --profile --json > bundle-stats.json; webpack-bundle-analyzer bundle-stats.json dist",
     "clean": "rm -rf dist junit bundle-stats.json",
-    "lint:js": "eslint ./*.js js/**/*.{js,vue} js-tests",
+    "lint:js": "eslint ./*.js js js/**/*.{js,vue} js-tests",
```

Comment:
The old glob pattern skipped files in the js/ directory itself.

Reply:
I would not have expected that.

Reply:
Me either. I only found it because I modified `main.js` to test ESLint, and it didn't find the error. It's really surprising behavior.